pages/Documents/MessagingChannels/ConnectToMessaging/api.md

```diff
@@ -0,0 +1,254 @@
+| 500 | 100 | Internal Server Error | Database connection error |
+| 500 | 101 | Internal Server Error | Database query error |
+| 500 | 110 | Internal Server Error | Error fetching the handoff |
+| 500 | 130 | Internal Server Error | Error fetching the account setting |
+
+
+### Frequently Asked Questions
+
+<strong>What is the rate limit for the API?</strong>
+The current rate limit is 30 TPS/second/brand per API.
+
+<strong>Which channels are supported as of now?</strong>
+C2M supports SMS-Twilio and WA channels.
+
+<strong>Is there a throughput limitation for the data that gets passed from Twilio to LP? For example, if brand sends 100 Twilio msgs/sec (their max throughput), then can the data flow through to LP at the same rate?</strong>
+- C2M does not have any limitations on the message size while sending messages to twilio or other channels. However a large message may translate to more than one message when the recipient receives it.
+- Example: A message of more than 140 characters will be divided into two messages and sent to recipients.
+ 
+<strong>What is the lifespan of the app JWT? When we do need to get a new JWT, do we have to first make the call to LivePerson Domain API in order to get the sentinel service domain, or is that domain consistent enough that we can hard code that in?</strong>
+- An APP JWT expiration time is 1 hour from the time it is created. To get an app JWT from sentinel API, a call to domain api has to be made to get the sentinel api domain. This domain can be cached for some duration. We expect the domain to change in very rare cases. It’s still recommended that cache duration should not be more than 1 day.
+
+- When using app JWT to call C2M API, a response below indicates the jwt is expired and new app jwt to be obtained from sentinel api.
+	```json
+	{
+		"code": 0,
+		"message": "jwt expired"
+	}
+	```
+
+<strong>Do we need any other JWT other than APP JWT e.g. Consumer JWT?</strong>
+C2M service does not create or consume consumer JWT or other JWT except APP JWT. C2M API consumes AppJWT created from provided clientId and Secret for authentication.
+
+<strong>What should the authentication header look like, is the bearer token the only thing required even in production usage? Do we need to include our ConsumerKey/Secret or our AccessToken/Secret that we use in the 1.0 API at all, or any other information?</strong>
+App Jwt will be consumed as Bearer Token. No other key, secret or token will be consumed by C2M Messaging api.
+
+<strong>How does C2M 2.0 api provide status of message e.g. success/failure of delivery ?</strong>
+The C2M 2.0 campaign api is asynchronous meaning that the success and failure of a message to a recipient is noted only when the recipient is picked from the C2M internal queue and a message is sent as per pre configured message rate. The [conversations](#conversations-api-example-request-and-response) api will provide the status of recipients tied to the campaign created.
```

Comment:
We will have the report API, link to that if the doc is live. If not, say stay tuned?